src/repositories/friendship.repository.ts

```diff
@@ -66,4 +72,33 @@ export class FriendshipRepository extends Repository {
     });
     return body.friendship_status;
   }
+
+  async setBesties(input: SetBestiesInput = {}) {
+    const { body } = await this.client.request.send<FriendshipRepositorySetBestiesResponseRootObject>({
+      url: `/api/v1/friendships/set_besties/`,
+      method: 'POST',
+      form: this.client.request.sign({
+        _csrftoken: this.client.state.cookieCsrfToken,
+        _uid: this.client.state.cookieUserId,
+        device_id: this.client.state.deviceId,
+        _uuid: this.client.state.uuid,
+        module: 'favorites_home_list',
+        source: 'audience_manager',
+        add: this.validateSetBestiesArray(input.add),
```

Comment:
`input.add` equals either `undefined` or `Array<string | number>` according to `SetBestiesInput`, so no need to validate it, especially that way.
if the request doesn't fail, just leave it undefined, if it does - add a default. But don't validate it.

Reply:
It doesn't throw an error with missing add/remove, so I removed the validation and the default